[Thu Mar 03 01:52:22.982875 2022] [win32:error] [pid 6472:tid 776] (70014)End of file found: [client 127.0.0.1:61027] AH02101: Failed to read cgi file D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/index.py for testing
[Thu Mar 03 01:52:22.982875 2022] [cgi:error] [pid 6472:tid 776] (70014)End of file found: [client 127.0.0.1:61027] AH01222: don't know how to spawn child process: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/index.py
[Thu Mar 03 01:55:46.870843 2022] [win32:error] [pid 6472:tid 776] [client 127.0.0.1:61031] AH02102: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/index.py is not executable; ensure interpreted scripts have "#!" or "'!" first line
[Thu Mar 03 01:55:46.870843 2022] [cgi:error] [pid 6472:tid 776] (9)Bad file descriptor: [client 127.0.0.1:61031] AH01222: don't know how to spawn child process: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/index.py
[Thu Mar 03 01:56:24.367031 2022] [cgi:error] [pid 6472:tid 776] [client 127.0.0.1:61035] End of script output before headers: index.py
[Sat Mar 05 19:27:34.990843 2022] [ssl:warn] [pid 4760:tid 680] AH01909: localhost:443:0 server certificate does NOT include an ID which matches the server name
[Sat Mar 05 19:27:35.020971 2022] [ssl:warn] [pid 4760:tid 680] AH01909: localhost:443:0 server certificate does NOT include an ID which matches the server name
[Sat Mar 05 19:27:35.151029 2022] [mpm_winnt:notice] [pid 4760:tid 680] AH00455: Apache/2.4.52 (Win64) OpenSSL/1.1.1m PHP/8.1.2 configured -- resuming normal operations
[Sat Mar 05 19:27:35.151029 2022] [mpm_winnt:notice] [pid 4760:tid 680] AH00456: Apache Lounge VS16 Server built: Dec 17 2021 10:17:38
[Sat Mar 05 19:27:35.151029 2022] [core:notice] [pid 4760:tid 680] AH00094: Command line: 'D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\bin\\httpd.exe -d D:/Bitnami/wampstack-8.1.2-0/apache2 -f D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\conf\\httpd.conf'
[Sat Mar 05 19:27:35.151029 2022] [mpm_winnt:notice] [pid 4760:tid 680] AH00418: Parent: Created child process 6532
[Sat Mar 05 19:27:35.627592 2022] [ssl:warn] [pid 6532:tid 684] AH01909: localhost:443:0 server certificate does NOT include an ID which matches the server name
[Sat Mar 05 19:27:35.652446 2022] [ssl:warn] [pid 6532:tid 684] AH01909: localhost:443:0 server certificate does NOT include an ID which matches the server name
[Sat Mar 05 19:27:35.685568 2022] [mpm_winnt:notice] [pid 6532:tid 684] AH00354: Child: Starting 64 worker threads.
[Sun Mar 06 05:18:12.120398 2022] [mpm_winnt:notice] [pid 4760:tid 680] AH00422: Parent: Received shutdown signal -- Shutting down the server.
[Sun Mar 06 05:18:14.163429 2022] [mpm_winnt:notice] [pid 6532:tid 684] AH00364: Child: All worker threads have exited.
[Sun Mar 06 05:18:14.190495 2022] [mpm_winnt:notice] [pid 4760:tid 680] AH00430: Parent: Child process 6532 exited successfully.
[Sun Mar 06 20:18:03.367727 2022] [ssl:warn] [pid 4436:tid 704] AH01909: localhost:443:0 server certificate does NOT include an ID which matches the server name
[Sun Mar 06 20:18:03.410442 2022] [ssl:warn] [pid 4436:tid 704] AH01909: localhost:443:0 server certificate does NOT include an ID which matches the server name
[Sun Mar 06 20:18:03.550721 2022] [mpm_winnt:notice] [pid 4436:tid 704] AH00455: Apache/2.4.52 (Win64) OpenSSL/1.1.1m PHP/8.1.2 configured -- resuming normal operations
[Sun Mar 06 20:18:03.550721 2022] [mpm_winnt:notice] [pid 4436:tid 704] AH00456: Apache Lounge VS16 Server built: Dec 17 2021 10:17:38
[Sun Mar 06 20:18:03.550721 2022] [core:notice] [pid 4436:tid 704] AH00094: Command line: 'D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\bin\\httpd.exe -d D:/Bitnami/wampstack-8.1.2-0/apache2 -f D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\conf\\httpd.conf'
[Sun Mar 06 20:18:03.558759 2022] [mpm_winnt:notice] [pid 4436:tid 704] AH00418: Parent: Created child process 7140
[Sun Mar 06 20:18:04.048232 2022] [ssl:warn] [pid 7140:tid 500] AH01909: localhost:443:0 server certificate does NOT include an ID which matches the server name
[Sun Mar 06 20:18:04.063859 2022] [ssl:warn] [pid 7140:tid 500] AH01909: localhost:443:0 server certificate does NOT include an ID which matches the server name
[Sun Mar 06 20:18:04.095106 2022] [mpm_winnt:notice] [pid 7140:tid 500] AH00354: Child: Starting 64 worker threads.
[Sun Mar 06 20:52:33.287135 2022] [cgi:error] [pid 7140:tid 1268] [client 127.0.0.1:61103] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/index.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 20:52:33.287135 2022] [cgi:error] [pid 7140:tid 1268] [client 127.0.0.1:61103] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\index.py", line 18, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/index.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 20:52:33.287135 2022] [cgi:error] [pid 7140:tid 1268] [client 127.0.0.1:61103] AH01215:     with open('data/' + page_id, 'r') as f:\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/index.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 20:52:33.287135 2022] [cgi:error] [pid 7140:tid 1268] [client 127.0.0.1:61103] AH01215: FileNotFoundError: [Errno 2] No such file or directory: 'data/form'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/index.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 21:20:22.913316 2022] [cgi:error] [pid 7140:tid 1268] [client 127.0.0.1:61948] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/index.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 21:20:22.913316 2022] [cgi:error] [pid 7140:tid 1268] [client 127.0.0.1:61948] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\index.py", line 18, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/index.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 21:20:22.913316 2022] [cgi:error] [pid 7140:tid 1268] [client 127.0.0.1:61948] AH01215:     with open('data/' + page_id, 'r') as f:\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/index.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 21:20:22.913316 2022] [cgi:error] [pid 7140:tid 1268] [client 127.0.0.1:61948] AH01215: FileNotFoundError: [Errno 2] No such file or directory: 'data/form'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/index.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 21:35:58.242231 2022] [mpm_winnt:notice] [pid 4436:tid 704] AH00422: Parent: Received shutdown signal -- Shutting down the server.
[Sun Mar 06 21:36:00.256781 2022] [mpm_winnt:notice] [pid 7140:tid 500] AH00364: Child: All worker threads have exited.
[Sun Mar 06 21:36:00.303842 2022] [mpm_winnt:notice] [pid 4436:tid 704] AH00430: Parent: Child process 7140 exited successfully.
[Sun Mar 06 21:36:03.748907 2022] [ssl:warn] [pid 11832:tid 624] AH01909: localhost:443:0 server certificate does NOT include an ID which matches the server name
[Sun Mar 06 21:36:03.790294 2022] [ssl:warn] [pid 11832:tid 624] AH01909: localhost:443:0 server certificate does NOT include an ID which matches the server name
[Sun Mar 06 21:36:04.038594 2022] [mpm_winnt:notice] [pid 11832:tid 624] AH00455: Apache/2.4.52 (Win64) OpenSSL/1.1.1m PHP/8.1.2 configured -- resuming normal operations
[Sun Mar 06 21:36:04.038594 2022] [mpm_winnt:notice] [pid 11832:tid 624] AH00456: Apache Lounge VS16 Server built: Dec 17 2021 10:17:38
[Sun Mar 06 21:36:04.038594 2022] [core:notice] [pid 11832:tid 624] AH00094: Command line: 'D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\bin\\httpd.exe -d D:/Bitnami/wampstack-8.1.2-0/apache2 -f D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\conf\\httpd.conf'
[Sun Mar 06 21:36:04.046622 2022] [mpm_winnt:notice] [pid 11832:tid 624] AH00418: Parent: Created child process 5244
[Sun Mar 06 21:36:04.895190 2022] [ssl:warn] [pid 5244:tid 592] AH01909: localhost:443:0 server certificate does NOT include an ID which matches the server name
[Sun Mar 06 21:36:04.925042 2022] [ssl:warn] [pid 5244:tid 592] AH01909: localhost:443:0 server certificate does NOT include an ID which matches the server name
[Sun Mar 06 21:36:04.986535 2022] [mpm_winnt:notice] [pid 5244:tid 592] AH00354: Child: Starting 64 worker threads.
[Sun Mar 06 21:43:13.775833 2022] [cgi:error] [pid 5244:tid 1208] [client 127.0.0.1:55286] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/index.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 21:43:13.775833 2022] [cgi:error] [pid 5244:tid 1208] [client 127.0.0.1:55286] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\index.py", line 18, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/index.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 21:43:13.775833 2022] [cgi:error] [pid 5244:tid 1208] [client 127.0.0.1:55286] AH01215:     with open('data/' + page_id, 'r') as f:\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/index.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 21:43:13.775833 2022] [cgi:error] [pid 5244:tid 1208] [client 127.0.0.1:55286] AH01215: FileNotFoundError: [Errno 2] No such file or directory: 'data/form'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/index.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 23:06:24.810421 2022] [cgi:error] [pid 5244:tid 816] [client 127.0.0.1:56871] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/index.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 23:06:24.810973 2022] [cgi:error] [pid 5244:tid 816] [client 127.0.0.1:56871] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\index.py", line 18, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/index.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 23:06:24.810973 2022] [cgi:error] [pid 5244:tid 816] [client 127.0.0.1:56871] AH01215:     with open('data/' + page_id, 'r') as f:\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/index.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 23:06:24.810973 2022] [cgi:error] [pid 5244:tid 816] [client 127.0.0.1:56871] AH01215: FileNotFoundError: [Errno 2] No such file or directory: 'data/form'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/index.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 23:06:35.862730 2022] [cgi:error] [pid 5244:tid 816] [client 127.0.0.1:56872] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/index.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 23:06:35.862730 2022] [cgi:error] [pid 5244:tid 816] [client 127.0.0.1:56872] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\index.py", line 18, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/index.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 23:06:35.862730 2022] [cgi:error] [pid 5244:tid 816] [client 127.0.0.1:56872] AH01215:     with open('data/' + page_id, 'r') as f:\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/index.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 23:06:35.862730 2022] [cgi:error] [pid 5244:tid 816] [client 127.0.0.1:56872] AH01215: FileNotFoundError: [Errno 2] No such file or directory: 'data/form'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/index.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 23:06:46.761792 2022] [cgi:error] [pid 5244:tid 816] [client 127.0.0.1:56876] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 23:06:46.761792 2022] [cgi:error] [pid 5244:tid 816] [client 127.0.0.1:56876] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form.py", line 48, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 23:06:46.761792 2022] [cgi:error] [pid 5244:tid 816] [client 127.0.0.1:56876] AH01215:     '''.format(char_name=page_id, CharList=CharList, desc=description))\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 23:06:46.761792 2022] [cgi:error] [pid 5244:tid 816] [client 127.0.0.1:56876] AH01215: NameError: name 'page_id' is not defined\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 23:08:11.075003 2022] [cgi:error] [pid 5244:tid 816] [client 127.0.0.1:56883] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 23:08:11.075003 2022] [cgi:error] [pid 5244:tid 816] [client 127.0.0.1:56883] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form.py", line 41, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 23:08:11.075003 2022] [cgi:error] [pid 5244:tid 816] [client 127.0.0.1:56883] AH01215:     '''.format(char_name=page_id, CharList=CharList, desc=description))\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 23:08:11.075003 2022] [cgi:error] [pid 5244:tid 816] [client 127.0.0.1:56883] AH01215: NameError: name 'page_id' is not defined\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/index.py
[Sun Mar 06 23:11:14.188625 2022] [cgi:error] [pid 5244:tid 816] [client 127.0.0.1:56896] End of script output before headers: form.py, referer: http://127.0.0.1/form.py
[Sun Mar 06 23:11:14.188649 2022] [cgi:error] [pid 5244:tid 816] [client 127.0.0.1:56896] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form.py", line 41\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Sun Mar 06 23:11:14.188649 2022] [cgi:error] [pid 5244:tid 816] [client 127.0.0.1:56896] AH01215:     \r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Sun Mar 06 23:11:14.188649 2022] [cgi:error] [pid 5244:tid 816] [client 127.0.0.1:56896] AH01215:     ^\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Sun Mar 06 23:11:14.188649 2022] [cgi:error] [pid 5244:tid 816] [client 127.0.0.1:56896] AH01215: SyntaxError: unexpected EOF while parsing\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Sun Mar 06 23:30:49.464069 2022] [win32:error] [pid 5244:tid 816] (70014)End of file found: [client 127.0.0.1:56978] AH02101: Failed to read cgi file D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py for testing, referer: http://127.0.0.1/form.py
[Sun Mar 06 23:30:49.464069 2022] [cgi:error] [pid 5244:tid 816] (70014)End of file found: [client 127.0.0.1:56978] AH01222: don't know how to spawn child process: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py, referer: http://127.0.0.1/form.py
[Mon Mar 07 03:26:45.962564 2022] [mpm_winnt:notice] [pid 11832:tid 624] AH00422: Parent: Received shutdown signal -- Shutting down the server.
[Mon Mar 07 03:26:47.980309 2022] [mpm_winnt:notice] [pid 5244:tid 592] AH00364: Child: All worker threads have exited.
[Mon Mar 07 03:26:48.011558 2022] [mpm_winnt:notice] [pid 11832:tid 624] AH00430: Parent: Child process 5244 exited successfully.
[Mon Mar 07 10:20:26.087008 2022] [ssl:warn] [pid 4444:tid 680] AH01909: localhost:443:0 server certificate does NOT include an ID which matches the server name
[Mon Mar 07 10:20:26.117126 2022] [ssl:warn] [pid 4444:tid 680] AH01909: localhost:443:0 server certificate does NOT include an ID which matches the server name
[Mon Mar 07 10:20:26.278269 2022] [mpm_winnt:notice] [pid 4444:tid 680] AH00455: Apache/2.4.52 (Win64) OpenSSL/1.1.1m PHP/8.1.2 configured -- resuming normal operations
[Mon Mar 07 10:20:26.278269 2022] [mpm_winnt:notice] [pid 4444:tid 680] AH00456: Apache Lounge VS16 Server built: Dec 17 2021 10:17:38
[Mon Mar 07 10:20:26.278269 2022] [core:notice] [pid 4444:tid 680] AH00094: Command line: 'D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\bin\\httpd.exe -d D:/Bitnami/wampstack-8.1.2-0/apache2 -f D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\conf\\httpd.conf'
[Mon Mar 07 10:20:26.301751 2022] [mpm_winnt:notice] [pid 4444:tid 680] AH00418: Parent: Created child process 7044
[Mon Mar 07 10:20:26.771282 2022] [ssl:warn] [pid 7044:tid 540] AH01909: localhost:443:0 server certificate does NOT include an ID which matches the server name
[Mon Mar 07 10:20:26.795986 2022] [ssl:warn] [pid 7044:tid 540] AH01909: localhost:443:0 server certificate does NOT include an ID which matches the server name
[Mon Mar 07 10:20:26.830682 2022] [mpm_winnt:notice] [pid 7044:tid 540] AH00354: Child: Starting 64 worker threads.
[Tue Mar 08 01:57:50.608307 2022] [win32:error] [pid 7044:tid 1120] (70014)End of file found: [client 127.0.0.1:51944] AH02101: Failed to read cgi file D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py for testing, referer: http://127.0.0.1/form.py
[Tue Mar 08 01:57:50.608307 2022] [cgi:error] [pid 7044:tid 1120] (70014)End of file found: [client 127.0.0.1:51944] AH01222: don't know how to spawn child process: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py, referer: http://127.0.0.1/form.py
[Tue Mar 08 01:58:47.321783 2022] [win32:error] [pid 7044:tid 1120] (70014)End of file found: [client 127.0.0.1:51949] AH02101: Failed to read cgi file D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py for testing, referer: http://127.0.0.1/form.py
[Tue Mar 08 01:58:47.321783 2022] [cgi:error] [pid 7044:tid 1120] (70014)End of file found: [client 127.0.0.1:51949] AH01222: don't know how to spawn child process: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py, referer: http://127.0.0.1/form.py
[Tue Mar 08 02:00:31.782433 2022] [win32:error] [pid 7044:tid 1120] (70014)End of file found: [client 127.0.0.1:51968] AH02101: Failed to read cgi file D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py for testing, referer: http://127.0.0.1/form.py
[Tue Mar 08 02:00:31.782433 2022] [cgi:error] [pid 7044:tid 1120] (70014)End of file found: [client 127.0.0.1:51968] AH01222: don't know how to spawn child process: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py, referer: http://127.0.0.1/form.py
[Tue Mar 08 02:06:48.937089 2022] [mpm_winnt:notice] [pid 4444:tid 680] AH00422: Parent: Received shutdown signal -- Shutting down the server.
[Tue Mar 08 02:06:50.972830 2022] [mpm_winnt:notice] [pid 7044:tid 540] AH00364: Child: All worker threads have exited.
[Tue Mar 08 02:06:50.999860 2022] [mpm_winnt:notice] [pid 4444:tid 680] AH00430: Parent: Child process 7044 exited successfully.
[Tue Mar 08 08:57:50.263299 2022] [ssl:warn] [pid 4556:tid 552] AH01909: localhost:443:0 server certificate does NOT include an ID which matches the server name
[Tue Mar 08 08:57:50.304765 2022] [ssl:warn] [pid 4556:tid 552] AH01909: localhost:443:0 server certificate does NOT include an ID which matches the server name
[Tue Mar 08 08:57:50.476561 2022] [mpm_winnt:notice] [pid 4556:tid 552] AH00455: Apache/2.4.52 (Win64) OpenSSL/1.1.1m PHP/8.1.2 configured -- resuming normal operations
[Tue Mar 08 08:57:50.476561 2022] [mpm_winnt:notice] [pid 4556:tid 552] AH00456: Apache Lounge VS16 Server built: Dec 17 2021 10:17:38
[Tue Mar 08 08:57:50.476561 2022] [core:notice] [pid 4556:tid 552] AH00094: Command line: 'D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\bin\\httpd.exe -d D:/Bitnami/wampstack-8.1.2-0/apache2 -f D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\conf\\httpd.conf'
[Tue Mar 08 08:57:50.476561 2022] [mpm_winnt:notice] [pid 4556:tid 552] AH00418: Parent: Created child process 6544
[Tue Mar 08 08:57:50.987158 2022] [ssl:warn] [pid 6544:tid 652] AH01909: localhost:443:0 server certificate does NOT include an ID which matches the server name
[Tue Mar 08 08:57:51.014072 2022] [ssl:warn] [pid 6544:tid 652] AH01909: localhost:443:0 server certificate does NOT include an ID which matches the server name
[Tue Mar 08 08:57:51.048424 2022] [mpm_winnt:notice] [pid 6544:tid 652] AH00354: Child: Starting 64 worker threads.
[Wed Mar 09 23:30:26.093828 2022] [mpm_winnt:notice] [pid 4556:tid 552] AH00422: Parent: Received shutdown signal -- Shutting down the server.
[Wed Mar 09 23:30:28.141628 2022] [mpm_winnt:notice] [pid 6544:tid 652] AH00364: Child: All worker threads have exited.
[Wed Mar 09 23:30:28.183261 2022] [mpm_winnt:notice] [pid 4556:tid 552] AH00430: Parent: Child process 6544 exited successfully.
[Thu Mar 10 01:24:34.489793 2022] [ssl:warn] [pid 4816:tid 548] AH01909: localhost:443:0 server certificate does NOT include an ID which matches the server name
[Thu Mar 10 01:24:34.540011 2022] [ssl:warn] [pid 4816:tid 548] AH01909: localhost:443:0 server certificate does NOT include an ID which matches the server name
[Thu Mar 10 01:24:34.661087 2022] [mpm_winnt:notice] [pid 4816:tid 548] AH00455: Apache/2.4.52 (Win64) OpenSSL/1.1.1m PHP/8.1.2 configured -- resuming normal operations
[Thu Mar 10 01:24:34.661087 2022] [mpm_winnt:notice] [pid 4816:tid 548] AH00456: Apache Lounge VS16 Server built: Dec 17 2021 10:17:38
[Thu Mar 10 01:24:34.661087 2022] [core:notice] [pid 4816:tid 548] AH00094: Command line: 'D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\bin\\httpd.exe -d D:/Bitnami/wampstack-8.1.2-0/apache2 -f D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\conf\\httpd.conf'
[Thu Mar 10 01:24:34.661087 2022] [mpm_winnt:notice] [pid 4816:tid 548] AH00418: Parent: Created child process 4384
[Thu Mar 10 01:24:35.131901 2022] [ssl:warn] [pid 4384:tid 604] AH01909: localhost:443:0 server certificate does NOT include an ID which matches the server name
[Thu Mar 10 01:24:35.151989 2022] [ssl:warn] [pid 4384:tid 604] AH01909: localhost:443:0 server certificate does NOT include an ID which matches the server name
[Thu Mar 10 01:24:35.192259 2022] [mpm_winnt:notice] [pid 4384:tid 604] AH00354: Child: Starting 64 worker threads.
[Thu Mar 10 22:16:21.224485 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:52887] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 22:16:21.224485 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:52887] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form_answer.py", line 14, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 22:16:21.224485 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:52887] AH01215:     f.write(content)\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 22:16:21.224485 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:52887] AH01215: UnicodeEncodeError: 'cp949' codec can't encode character '\\ufffd' in position 0: illegal multibyte sequence\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 22:19:34.548984 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:52905] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 22:19:34.548984 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:52905] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form_answer.py", line 13, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 22:19:34.548984 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:52905] AH01215:     print(title, content)\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 22:19:34.548984 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:52905] AH01215: UnicodeEncodeError: 'cp949' codec can't encode character '\\ufffd' in position 0: illegal multibyte sequence\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 22:19:43.342590 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:52906] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 22:19:43.342590 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:52906] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form_answer.py", line 13, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 22:19:43.342590 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:52906] AH01215:     print(title, content)\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 22:19:43.352589 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:52906] AH01215: UnicodeEncodeError: 'cp949' codec can't encode character '\\ufffd' in position 0: illegal multibyte sequence\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 22:23:01.415676 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:52911] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 22:23:01.415676 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:52911] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form_answer.py", line 13, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 22:23:01.415676 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:52911] AH01215:     print(title, content)\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 22:23:01.415676 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:52911] AH01215: UnicodeEncodeError: 'cp949' codec can't encode character '\\ufffd' in position 0: illegal multibyte sequence\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 22:33:06.985228 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:52937] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 22:33:06.985228 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:52937] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form_answer.py", line 15, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 22:33:06.985228 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:52937] AH01215:     print(title, content)\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 22:33:06.985228 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:52937] AH01215: UnicodeEncodeError: 'cp949' codec can't encode character '\\ufffd' in position 0: illegal multibyte sequence\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form_answer.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 22:47:50.866491 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53095] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 22:47:50.866491 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53095] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form.py", line 12, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 22:47:50.866491 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53095] AH01215:     formList = formList + f'<a href="index.py?id={item}" title="{item}"><li id="char">{item}</li></a>'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 22:47:50.866491 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53095] AH01215: NameError: name 'formList' is not defined\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 22:47:55.879655 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53095] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 22:47:55.879655 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53095] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form.py", line 12, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 22:47:55.879655 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53095] AH01215:     formList = formList + f'<a href="index.py?id={item}" title="{item}"><li id="char">{item}</li></a>'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 22:47:55.879655 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53095] AH01215: NameError: name 'formList' is not defined\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 22:49:07.613627 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53096] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 22:49:07.613627 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53096] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form.py", line 12, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 22:49:07.613627 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53096] AH01215:     formList = formList + f'<a href="form.py?id={item}" title="{item}"><li>{item}</li></a>'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 22:49:07.613627 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53096] AH01215: NameError: name 'formList' is not defined\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 22:57:02.352568 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53137] End of script output before headers: form.py, referer: http://127.0.0.1/form.py?id=new%20note
[Thu Mar 10 22:57:02.352568 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53137] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form.py", line 16\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new%20note
[Thu Mar 10 22:57:02.352568 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53137] AH01215:     content = '<p>'+ query_string['content'].value}+'</p>'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new%20note
[Thu Mar 10 22:57:02.352568 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53137] AH01215:                                                   ^\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new%20note
[Thu Mar 10 22:57:02.352568 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53137] AH01215: SyntaxError: unmatched '}'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new%20note
[Thu Mar 10 22:57:54.698670 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53140] End of script output before headers: form.py, referer: http://127.0.0.1/form.py?id=new%20note
[Thu Mar 10 22:57:54.698670 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53140] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form.py", line 16\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new%20note
[Thu Mar 10 22:57:54.698670 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53140] AH01215:     content = '<p>{query_string['content'].value}</p>'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new%20note
[Thu Mar 10 22:57:54.698670 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53140] AH01215:                                  ^\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new%20note
[Thu Mar 10 22:57:54.698670 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53140] AH01215: SyntaxError: invalid syntax\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new%20note
[Thu Mar 10 22:57:56.723689 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53141] End of script output before headers: form.py, referer: http://127.0.0.1/form.py?id=new%20note
[Thu Mar 10 22:57:56.723689 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53141] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form.py", line 16\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new%20note
[Thu Mar 10 22:57:56.723689 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53141] AH01215:     content = '<p>{query_string['content'].value}</p>'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new%20note
[Thu Mar 10 22:57:56.723689 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53141] AH01215:                                  ^\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new%20note
[Thu Mar 10 22:57:56.723689 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53141] AH01215: SyntaxError: invalid syntax\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new%20note
[Thu Mar 10 22:59:36.353403 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53148] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 22:59:36.353403 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53148] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form.py", line 16, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 22:59:36.353403 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53148] AH01215:     content = '<p>'+ query_string['content'].value+'</p>'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 22:59:36.353403 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53148] AH01215:   File "D:\\anaconda3\\lib\\cgi.py", line 525, in __getitem__\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 22:59:36.353403 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53148] AH01215:     raise KeyError(key)\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 22:59:36.353403 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53148] AH01215: KeyError: 'content'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 22:59:37.114040 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53148] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 22:59:37.114040 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53148] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form.py", line 16, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 22:59:37.114040 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53148] AH01215:     content = '<p>'+ query_string['content'].value+'</p>'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 22:59:37.114040 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53148] AH01215:   File "D:\\anaconda3\\lib\\cgi.py", line 525, in __getitem__\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 22:59:37.114040 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53148] AH01215:     raise KeyError(key)\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 22:59:37.114040 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53148] AH01215: KeyError: 'content'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 22:59:40.168597 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53148] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 22:59:40.168597 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53148] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form.py", line 16, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 22:59:40.168597 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53148] AH01215:     content = '<p>'+ query_string['content'].value+'</p>'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 22:59:40.168597 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53148] AH01215:   File "D:\\anaconda3\\lib\\cgi.py", line 525, in __getitem__\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 22:59:40.168597 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53148] AH01215:     raise KeyError(key)\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 22:59:40.168597 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53148] AH01215: KeyError: 'content'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 23:00:01.257276 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53149] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 23:00:01.257276 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53149] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form.py", line 16, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 23:00:01.257276 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53149] AH01215:     content = '<p>'+query_string['content'].value+'</p>'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 23:00:01.257276 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53149] AH01215:   File "D:\\anaconda3\\lib\\cgi.py", line 525, in __getitem__\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 23:00:01.257276 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53149] AH01215:     raise KeyError(key)\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 23:00:01.257276 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53149] AH01215: KeyError: 'content'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 23:02:16.676133 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53166] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:02:16.676133 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53166] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form.py", line 17, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:02:16.676133 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53166] AH01215:     content = '<p>'+query_string["content"].value+'</p>'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:02:16.676133 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53166] AH01215:   File "D:\\anaconda3\\lib\\cgi.py", line 525, in __getitem__\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:02:16.676133 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53166] AH01215:     raise KeyError(key)\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:02:16.676133 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53166] AH01215: KeyError: 'content'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:03:02.188041 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53170] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:03:02.188041 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53170] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form.py", line 16, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:03:02.188041 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53170] AH01215:     cont = query_string["content"].value\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:03:02.188041 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53170] AH01215:   File "D:\\anaconda3\\lib\\cgi.py", line 525, in __getitem__\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:03:02.188041 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53170] AH01215:     raise KeyError(key)\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:03:02.188041 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53170] AH01215: KeyError: 'content'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:04:23.392366 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53173] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:04:23.392366 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53173] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form.py", line 15, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:04:23.392366 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53173] AH01215:     page_cont = query_string["content"].value\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:04:23.392366 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53173] AH01215:   File "D:\\anaconda3\\lib\\cgi.py", line 525, in __getitem__\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:04:23.392366 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53173] AH01215:     raise KeyError(key)\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:04:23.392366 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53173] AH01215: KeyError: 'content'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:04:26.140289 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53173] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 23:04:26.140289 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53173] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form.py", line 15, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 23:04:26.140289 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53173] AH01215:     page_cont = query_string["content"].value\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 23:04:26.140289 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53173] AH01215:   File "D:\\anaconda3\\lib\\cgi.py", line 525, in __getitem__\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 23:04:26.140289 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53173] AH01215:     raise KeyError(key)\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 23:04:26.140289 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53173] AH01215: KeyError: 'content'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py?id=new2
[Thu Mar 10 23:07:46.090247 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53183] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:07:46.090247 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53183] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form.py", line 50, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:07:46.090247 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53183] AH01215:     {content}\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:07:46.090247 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53183] AH01215: NameError: name 'content' is not defined\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:07:50.615340 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53183] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:07:50.615340 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53183] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form.py", line 50, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:07:50.615340 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53183] AH01215:     {content}\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:07:50.615340 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53183] AH01215: NameError: name 'content' is not defined\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:24:10.420515 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53287] End of script output before headers: form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:24:10.420515 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53287] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form.py", line 22\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:24:10.420515 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53287] AH01215:     else:\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:24:10.420515 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53287] AH01215:     ^\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:24:10.420515 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53287] AH01215: SyntaxError: invalid syntax\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Thu Mar 10 23:24:34.251037 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53289] AH02811: script not found or unable to stat: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/update.py, referer: http://127.0.0.1/form.py?id=new%20note
[Thu Mar 10 23:26:09.329338 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53294] AH02811: script not found or unable to stat: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/update.py, referer: http://127.0.0.1/form.py?id=new%20note
[Thu Mar 10 23:26:36.539084 2022] [cgi:error] [pid 4384:tid 620] [client 127.0.0.1:53298] AH02811: script not found or unable to stat: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/update.py, referer: http://127.0.0.1/form.py?id=new%20note
[Fri Mar 11 00:29:19.274360 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53623] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/update.py, referer: http://127.0.0.1/form.py?id=nb5
[Fri Mar 11 00:29:19.275361 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53623] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\update.py", line 14, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/update.py, referer: http://127.0.0.1/form.py?id=nb5
[Fri Mar 11 00:29:19.275361 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53623] AH01215:     pageId = query_string['id'].value\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/update.py, referer: http://127.0.0.1/form.py?id=nb5
[Fri Mar 11 00:29:19.275361 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53623] AH01215:   File "D:\\anaconda3\\lib\\cgi.py", line 525, in __getitem__\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/update.py, referer: http://127.0.0.1/form.py?id=nb5
[Fri Mar 11 00:29:19.275361 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53623] AH01215:     raise KeyError(key)\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/update.py, referer: http://127.0.0.1/form.py?id=nb5
[Fri Mar 11 00:29:19.275361 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53623] AH01215: KeyError: 'id'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/update.py, referer: http://127.0.0.1/form.py?id=nb5
[Fri Mar 11 00:29:36.240002 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53624] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/update.py, referer: http://127.0.0.1/form.py?id=nb5
[Fri Mar 11 00:29:36.240002 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53624] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\update.py", line 14, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/update.py, referer: http://127.0.0.1/form.py?id=nb5
[Fri Mar 11 00:29:36.240002 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53624] AH01215:     pageId = query_string['id'].value\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/update.py, referer: http://127.0.0.1/form.py?id=nb5
[Fri Mar 11 00:29:36.240002 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53624] AH01215:   File "D:\\anaconda3\\lib\\cgi.py", line 525, in __getitem__\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/update.py, referer: http://127.0.0.1/form.py?id=nb5
[Fri Mar 11 00:29:36.240002 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53624] AH01215:     raise KeyError(key)\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/update.py, referer: http://127.0.0.1/form.py?id=nb5
[Fri Mar 11 00:29:36.241087 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53624] AH01215: KeyError: 'id'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/update.py, referer: http://127.0.0.1/form.py?id=nb5
[Fri Mar 11 01:19:36.129226 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53955] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Fri Mar 11 01:19:36.129226 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53955] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form.py", line 33, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Fri Mar 11 01:19:36.129226 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53955] AH01215:     file_data = sanitizer.sanitize(file_data)\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Fri Mar 11 01:19:36.129226 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53955] AH01215: TypeError: sanitize() missing 1 required positional argument: 'html'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Fri Mar 11 01:19:40.041439 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53955] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Fri Mar 11 01:19:40.041439 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53955] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form.py", line 33, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Fri Mar 11 01:19:40.041439 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53955] AH01215:     file_data = sanitizer.sanitize(file_data)\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Fri Mar 11 01:19:40.041439 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53955] AH01215: TypeError: sanitize() missing 1 required positional argument: 'html'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Fri Mar 11 01:22:02.296324 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53965] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Fri Mar 11 01:22:02.296324 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53965] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form.py", line 33, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Fri Mar 11 01:22:02.296324 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53965] AH01215:     file_data = sanitizer.sanitize(file_data)\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Fri Mar 11 01:22:02.296324 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53965] AH01215: TypeError: sanitize() missing 1 required positional argument: 'html'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Fri Mar 11 01:22:04.601033 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53965] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Fri Mar 11 01:22:04.601033 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53965] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form.py", line 33, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Fri Mar 11 01:22:04.601033 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53965] AH01215:     file_data = sanitizer.sanitize(file_data)\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Fri Mar 11 01:22:04.601033 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53965] AH01215: TypeError: sanitize() missing 1 required positional argument: 'html'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Fri Mar 11 01:27:46.289165 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53977] AH01215: Traceback (most recent call last):\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Fri Mar 11 01:27:46.289165 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53977] AH01215:   File "D:\\Bitnami\\wampstack-8.1.2-0\\apache2\\htdocs\\form.py", line 33, in <module>\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Fri Mar 11 01:27:46.289165 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53977] AH01215:     file_data = sanitizer.sanitize(file_data)\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
[Fri Mar 11 01:27:46.289165 2022] [cgi:error] [pid 4384:tid 1056] [client 127.0.0.1:53977] AH01215: TypeError: sanitize() missing 1 required positional argument: 'html'\r: D:/Bitnami/wampstack-8.1.2-0/apache2/htdocs/form.py, referer: http://127.0.0.1/form.py
